src/tracks/playabletrack/wavetrack/ui/CutlineHandle.cpp

```diff
@@ -169,7 +169,16 @@ UIHandle::Result CutlineHandle::Click
 
          viewInfo.selectedRegion.setTimes(cutlineStart, cutlineEnd);
       }
-      else if (mLocation.typ == WaveTrackLocation::locationMergePoint) {
+      // todo(mhodgkinson) Bluntly disabling this Join shortcut for now. There
+      // exists an issue that wants to do this properly:
+      // https://github.com/audacity/audacity/issues/5020
```

Comment:
Replicated in another PR -- again, wrong number?

Reply:
Spot on.